### ISSUE TITLE
Duplicated entries with multiple Svelte components

### ISSUE THREAD

**Issue by trevyn** [windicss/svelte-windicss-preprocess#53]
Having multiple Svelte components appears to create duplicated `*`, `html`, `body` etc. entries in the resulting CSS:

<img src="https://user-images.githubusercontent.com/230691/110273019-131f7880-7fe5-11eb-9a60-8b2388afefcc.png" width="300">

**Comment by alexanderniebuhr** [comment]
please provide debug logs of svelte-windicss-preprocess, you can enable in config with
```
silent: false,
debug: true
```
also provide the version you are using of it according to the release tags of this repo. Which bundler do you use?

**Comment by trevyn** [comment]
Using snowpack with `svelte-windicss-preprocess` v3.0.0-beta.1

Where should I be looking for the logs? It doesn't show anything, and no extra files are created:

```
$ npm run build

> @ build /Users/eden/Desktop/turbo/frontend
> snowpack build

│
│ svelte-windicss-preprocess - v3.0.0-beta.1
│ 
│ - windicss running mode: prod
│ - advanced debug logs: on
│    • compilation mode: disabled
│    • class prefix: windi-
│    • global preflights: enabled
│    • global utilities: enabled
│
└──────────
[snowpack] ! building source files...
[snowpack] ✔ build complete [1.73s]
[snowpack] ! building dependencies...
[snowpack] ✔ dependencies ready! [4.50s]
[snowpack] ! verifying build...
[snowpack] ✔ verification complete [0.01s]
[snowpack] ! writing build to disk...
[snowpack] ! optimizing build...
[snowpack] ✔ optimize complete [0.35s]
[snowpack] ▶ Build Complete!
```

**Comment by alexanderniebuhr** [comment]
thanks, that was all I was looking for. Can you try to delete `globalPreflights & globalUtilities` from your preprocess config. Also are the styles duplicated in one css or are they existing in multiple files?

**Comment by trevyn** [comment]
Same result after deleting `globalPreflights` & `globalUtilities`. Styles are duplicated in one output css file, and it also happens when running in dev mode.

Will try to reproduce with example project now.

**Comment by alexanderniebuhr** [comment]
Ok, i asked you to reproduce with example project so I can test it myself. If that does not work, please put your project with this behavior on github, so I can clone and check :)

**Comment by trevyn** [comment]
Ahhh, I think I see what's happening. I want a single `.css` file output, so I'm using

```
 optimize: {
  bundle: true,
 },
```

in the `snowpack.config.js`, and I guess it's just running each Svelte component separately and then gluing all the CSS together. So I guess the best option is for me to figure out how to de-dupe the result...

Project is in the `frontend` directory of [trevyn/turbo](https://github.com/trevyn/turbo).

**Comment by alexanderniebuhr** [comment]
Svelte windicss preprocess has a hidden bundle config in v2, you can check if that works. V3 will support this too? Just not in beta 1

bundle: <FILEPATH>

**Comment by alexanderniebuhr** [comment]
Also svelte does de-dedup by default. So it might be snowpack problem here. I will check your project later tonight

**Comment by noahsalvi** [comment]
I also get this problem with rollup.

```
│ svelte-windicss-preprocess - v3.0.0-beta.3
│ NODE_ENV is undefined
│ - windicss running mode: prod
│ - advanced debug logs: on
│    • compilation mode: disabled
│    • class prefix: windi-
│    • global preflights: enabled
│    • global utilities: disabled
```

For every component in my app, there is a duplicate of windicss reset.

REPO:
1. Install official svelte template
2. Add svelte-windicss-preprocess
3. Add to rollup
4. Create a new component with tailwind classes
5. Import into the app component.
6. build the project
7. Check bundle.css

**Comment by alexanderniebuhr** [comment]
this should be fixed for bundlers which do not parsing svelte files in parallel. snowpack is known for doing it, so it will genereate mutliple css files. while rollup seems to generate one main css
